src/production.cpp

```diff
@@ -36,35 +37,40 @@ static void readButtons(TaskContext *ctx) {
 }
 
 String tag = "";
+char *endptr;
+int base10tag;
```

Comment:
Why are those global variables?
And I'm not sure `String tag` is still used.

Reply:
`String tag` is still used to parse the tag.